src/quo2/components/avatars/wallet_user_avatar.cljs

```diff
@@ -38,8 +38,8 @@
         l-name-initial (-> l-name
                            string/upper-case
                            (subs 0 1))
-        circle-color   (colors/custom-color color 50 20)
-        text-color     (colors/custom-color-by-theme color 50 60)]
+        circle-color   (colors/custom-color customization-color 50 20)
+        text-color     (colors/custom-color-by-theme customization-color 50 60)]
```

Comment:
nit this doesn't take theme prop, perhaps use
```
(colors/theme-colors 
(colors/custom-color customization-color 50)
(colors/custom-color customization-color 60)
theme)
```
going to properly fix this api soon! :)

Reply:
@J-Son89 This component is not using our `with-theme` fn, also it'd be good to refactor it's API sizes and the `circle-color` var is not changing according to the theme (it's always 50).

Instead of fixing it here, I created an issue:

https://github.com/status-im/status-mobile/issues/17481

Reply:
Sure sounds good to me. Thanks @ulisesmac!